lib/fluent/plugin/filter_record_modifier.rb

```diff
@@ -66,6 +71,20 @@ def configure(conf)
         end
       end
 
+      @replace_keys = Array.new
+      if @replace
+        conf.elements.select { |element| element.name == 'replace' }.each do |element|
```

Comment:
You can use `config_section` for it.
See rewrite-tag-filter code: https://github.com/fluent/fluent-plugin-rewrite-tag-filter/blob/9565ddf978faf0d419ffa3a796cbf8cbd292a9af/lib/fluent/plugin/out_rewrite_tag_filter.rb#L15

Reply:
Thx. I rewrote code with using `config_section`.